learn/releasenotes/7-16-0.adoc

```diff
@@ -0,0 +1,4 @@
+== jBPM Server Distribution context update
```

Comment:
I think we have much more to mention ... maybe take a look at jiras from last two sprints?

Reply:
@mswiderski sure, added a few more items.